dropwizard-oauth/src/main/java/smartthings/dw/oauth/OAuthToken.java

```diff
@@ -65,6 +72,7 @@ public String toString() {
 			.add("user", user)
 			.add("scopes", scopes)
 			.add("clientId", clientId)
+            .add("authToken", authToken)
```

Comment:
To prevent accidental logging of user's authentication token, we might not want to put this in a toString() method.